### ISSUE TITLE
Bug: getLocalBounds behaves differently in PixiJS v8.5.2 compared to v8.6.1

### ISSUE THREAD

**Issue by Serhiy-Y** [pixijs/pixijs#11105]
### Current Behavior

`sprite.getLocalBounds().width` returns wrong value

### Expected Behavior

`sprite.getLocalBounds().width` returns correct value

### Steps to Reproduce

I am creating a sprite from sprite sheet texture and difference in `sprite.getLocalBounds().width` huge 73 (v8.5.2) vs 44 (v8.6.1).

If you need a code example, please share the email address where I can send it (unfortunately, I cannot share private resources publicly).
I think it's related to #10971 

### Environment

- **`pixi.js` version**: *8.6.1*

**Comment by bigtimebuddy** [comment]
Can you please provide a minimal reproduction? Thanks.

**Comment by Serhiy-Y** [comment]
> Can you please provide a minimal reproduction? Thanks.

yes, can you share the email please, because it contains private resources.

**Comment by bigtimebuddy** [comment]
It's the most helpful if you recreate the bug with the smallest amount of code using non-private assets with web tools like pixiplayground.com, jsfiddle.net or codesandbox.com?

**Comment by SerG-Y** [comment]
Sure, it seems easier than I thought. I just used the sprite sheet from the PixiJS tests :)

Example: https://codesandbox.io/p/sandbox/wwjfrj

**Comment by SerG-Y** [comment]
Thanks a lot for the fix!
